frappe/public/less/mobile.less

```diff
@@ -228,9 +223,9 @@ body {
 				width: calc(~"100% - 90px");
 			}
 			.indicator {
-				float: left;
-				margin-top: 10px;
-				margin-right: 5px;
+				position: absolute;
```

Comment:
Instead of using absolute positioning here
Add a new class to the following snippet in `page.html`

```
<div class="flex fill-width <new-class>">
	<div class="ellipsis title-text"></div>
	<span class="indicator whitespace-nowrap hide"></span>
</div>
```

And for that new-class set `flex-direction: column` when the screen size is mobile.